internal/controllers/apis/configmap/disable.go

```diff
@@ -50,10 +65,15 @@ func MaybeNewDisableStatus(ctx context.Context, client client.Client, disableSou
 		return cm, err
 	}
 
-	isDisabled, reason, err := DisableStatus(getCM, disableSource)
+	result, reason, err := DisableStatus(getCM, disableSource)
```

Comment:
i feel like having both a DisableResultError state and an `err` state - but they have totally different behavior - is a recipe for problems

Reply:
nm this comment, i was wrong, i misread where the error was coming from